src/Options.jl

```diff
@@ -22,8 +24,8 @@ end
   All types for this purpose have this abstract type as super-type.
   
   Required fields are: `name`, `lastchanged`, `options`
-  """
-abstract AbstractOptionsODE <: Any
+"""
+AbstractOptionsODE
```

Comment:
What's with the abstract types? You should `@compat abstract type name end` and require Compat 0.17.0